.github/workflows/semantic.yml

```diff
@@ -0,0 +1,33 @@
+# Check that the commit messages are between 20 and 150 chars,
+# and that they follow the rule
+# <type> (<scope>): <subject>
+# type: docs, feat, fix, refactor, style or test
+# scope (optional): any extra info, (like DMS or whatever)
+
+name: 'Commit Message Check'
+on: pull_request
+
+jobs:
+  check-commit-message:
```

Comment:
The reason we didn't have this was that it can probably be done with pre-commit.

Also, this PR feels like many merged into one.

Reply:
it's actually going to be driven by https://github.com/DIRACGrid/diracx/issues/264

Reply:
Correct, removed.